test/main/java/com/liveramp/hyperminhash/TestBetaMinHashSerde.java

```diff
@@ -0,0 +1,58 @@
+package com.liveramp.hyperminhash;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.Random;
+
+public class TestBetaMinHashSerde {
+
+  @Test
+  public void testRoundtripEmptySketch() {
+    BetaMinHashSerde serde = new BetaMinHashSerde();
+    int iterations = 1_000;
+
+    RandomTestRunner.runRandomizedTest(iterations, rng -> {
+      BetaMinHash sketch = new BetaMinHash();
```

Comment:
Do we get anything out of performing this test 1000 times if an empty sketch is always the same?

Reply:
great point. We do not.